testdata/flushable_ingest

```diff
@@ -618,3 +618,70 @@ L0.1:
   000004:[a#12,SET-b#12,SET]
 L0.0:
   000007:[a#10,SET-b#11,SET]
+
+close
+----
+
+reset
+----
+
+build small
+set-multiple 10 small
+----
+
+build large
+set-multiple 100000 large
+----
+
+batch
+set small-00001 before-ingest
+set large-00001 before-ingest
+----
+
+blockFlush
```

Comment:
nit: maybe the blockFlush can go before the batch commit, that way we are guaranteed we wouldn't flush right after the batch?